### PR TITLE
fix: Use alias schema

### PR DESCRIPTION
**Pull request by ChaituVR**
Merge after updating snapshot.js version https://github.com/snapshot-labs/snapshot.js/pull/913

### CODE REVIEW

File: src/writer/alias.ts [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+import { jsonParse } from '../helpers/utils';
+import log from '../helpers/log';
 
 export async function verify(message): Promise<any> {
-  return message.from !== message.alias;
+  const msg = jsonParse(message.msg, {});
+  const schemaIsValid = snapshot.utils.validateSchema(snapshot.schemas.alias, msg.payload);
+  if (schemaIsValid !== true) {
+    log.warn(`[writer] Wrong statement format ${JSON.stringify(schemaIsValid)}`);
+    return Promise.reject('wrong statement format');
```
Comment (wa0x6e) [inline src/writer/alias.ts:12]:
```suggestion
    return Promise.reject('wrong alias format');
```

File: src/writer/alias.ts [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 import db from '../helpers/mysql';
+import { jsonParse } from '../helpers/utils';
+import log from '../helpers/log';
 
 export async function verify(message): Promise<any> {
-  return message.from !== message.alias;
+  const msg = jsonParse(message.msg, {});
+  const schemaIsValid = snapshot.utils.validateSchema(snapshot.schemas.alias, msg.payload);
+  if (schemaIsValid !== true) {
+    log.warn(`[writer] Wrong statement format ${JSON.stringify(schemaIsValid)}`);
```
Comment (wa0x6e) [inline src/writer/alias.ts:11]:
```suggestion
    log.warn(`[writer] Wrong alias format ${JSON.stringify(schemaIsValid)}`);
```

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: src/writer/alias.ts [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+    return Promise.reject('wrong statement format');
+  }
+  return message.from !== msg.payload.alias;
```
Comment (wa0x6e) [inline src/writer/alias.ts:14]:
```suggestion
  if (message.from === msg.payload.alias) {
    return Promise.reject('alias cannot be the same as the address');
  }
```

**Review by wa0x6e — CHANGES_REQUESTED** [review]
`verify` expect a Promise.rejection on error

**Comment by wa0x6e** [comment]
I've also commit some tests

File: src/writer/alias.ts (hunk 3 above)
Comment (ChaituVR) [reply src/writer/alias.ts:14]:
```suggestion
  if (message.from === msg.payload.alias) {
	    return Promise.reject('alias cannot be the same as the address');
	}
  return true;
```

**Comment by wa0x6e** [comment]
Lint error coming from suggested changes 🙈

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**